Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonThemeBrowserForm.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Krypton.Toolkit
 {
-    public partial class KryptonThemeBrowserForm : KryptonForm
+    internal partial class KryptonThemeBrowserForm : KryptonForm
```

Comment:
This stops it from being used by external Users of the Suite. Is that the intention ?

Reply:
I was going to make a public API for it via `KryptonThemeBrowser`, but the theme index numbers seem to be broken

Reply:
Reverted changes